packages/plugins/data-access/coco-data-access/src/index.ts

```diff
@@ -40,6 +42,11 @@ class DataLoader implements ImageDataLoader {
       label: this.dataPairs[id].label
     };
   }
+
+  async setItem(id: number, sample: ImageSample) {
```

Comment:
```suggestion
  setItem(id: number, sample: ImageSample): void {
```

Reply:
This function is extended from super-class async function. Acutally return value is Promise<void>. I give it as async function since it's likely that in some cases we need to do some async logic inside setItem

Reply:
If the return value is void, we need to use `: Promise<void>`. 
```
The second part is the return type. We make it clear which is the return type by using an arrow (=>) between the parameters and the return type. As mentioned before, this is a required part of the function type, so if the function doesn’t return a value, you would use void instead of leaving it off.
```
https://www.typescriptlang.org/docs/handbook/functions.html#writing-the-function-type